examples/pxScene2d/src/CMakeLists.txt

```diff
@@ -468,6 +469,10 @@ if (BUILD_WITH_TEXTURE_USAGE_MONITORING)
     set(PXSCENE_DEFINITIONS ${PXSCENE_DEFINITIONS} -DENABLE_PX_SCENE_TEXTURE_USAGE_MONITORING)
 endif (BUILD_WITH_TEXTURE_USAGE_MONITORING)
 
+if (BUILD_WITH_PXOBJECT_TRACKING)
```

Comment:
Let's add a message() statement here to indicate we are building with object tracking enabled.

Reply:
added message